feature_engine/mathematical_transformers.py

```diff
@@ -0,0 +1,86 @@
+from feature_engine.base_transformers import BaseNumericalTransformer
+
+
+class AdditionTransformer(BaseNumericalTransformer):
+    """
+    The AdditionTransfomer() applies basic mathematical operations using aggregation of features.
+
+    Parameters
+    ----------
+
+    variables: list, default=None
+        The list of numerical variables to be transformed. If None, the transformer
+        will find and select all numerical variables.
+
+    math_operations: list, default=None
+        The list of basic math operations to be used in transformation.
+
+        Each operation should be a string and must be one of elements
+        from list: ['sum', 'prod', 'mean', 'std', 'max', 'min']
+
+        Each operation will result in operation column in result dataset.
+
+        If None, the transformer will calculate each of permitted
+        operations: ['sum', 'prod', 'mean', 'std', 'max', 'min']
+    """
+
+    def __init__(self, variables=None, math_operations=None):
+        self.variables = variables
+        self.math_operations = math_operations
```

Comment:
here I would put something like:
```
        if isinstance(math_operations, list):
            if any(elem_par not in self.math_operations_permitted for elem_par in self.math_operations):
                raise KeyError("At least one of math_operations is not found in permitted operations set. "
                               "Choose one of ['sum', 'prod', 'mean', 'std', 'max', 'min']")
            self.math_operations = math_operations
```
which is what you wrote below but here

Reply:
thx, very good point!